nitrous-post-create.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# KeystoneJS Demo App - Nitrous QuiskStart post create script by @tmvanetten
+
+#KeystoneJS Demo App is used IAW https://github.com/keystonejs/keystone-demo#license
```

Comment:
In Accordance With.  Sorry for the acronym..old habit from being in the military.